sale_order_secondary_unit/tests/test_sale_order_secondary_unit.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import SavepointCase
+
+
+class TestSaleOrderSecondaryUnit(SavepointCase):
+    at_install = False
```

Comment:
@digitalsatori you have to put this as decorators instead of variables, as in v12 this is not supported this way:
```python
from odoo.tests import tagged

@tagged('post_install', '-at_install')
class TestSaleOrderSecondaryUnit(SavepointCase):
    ...
```

Reply:
👍 awesome, will do as instructed .

Reply:
@pedrobaeza Is mandatory? I think that to pass tests in OCA is not needed...